src/IndexService/Config/AbstractConfig.php

```diff
@@ -175,13 +175,8 @@ public function getTenantName(): string
      *
      * @return Attribute[]
```

Comment:
```suggestion
     * @return Attribute[]|null
```

or remove the php doc because its already defined in the interface.

Reply:
I thought about that, but tbh I don't want to start en remove Phpdoc only for this one. It would only make sense to do it as a whole.

Reply:
Than you should add the `null` here also